ui/pages/confirmations/confirmation/templates/add-ethereum-chain.js

```diff
@@ -244,6 +244,109 @@ function getValues(pendingApproval, t, actions, history, data) {
   return {
     content: [
       {
+<<<<<<< HEAD
+=======
```

Comment:
Some merge conflict cruft still here that you could probably remove

Reply:
Noted!